.buildkite/Dockerfile

```diff
@@ -1,27 +1,20 @@
 FROM ubuntu:18.04
 
-EXPOSE 22222
-
-RUN apt update
-RUN apt install -y iproute2 tinysshd
-RUN apt clean
-
-RUN mkdir -p /root/.ssh
-COPY .buildkite/config/ubuntu.pubkey /root/.ssh/authorized_keys
-RUN chmod 0600 /root/.ssh/authorized_keys
-RUN chmod 0700 /root/.ssh
-COPY .buildkite/config/tinysshd.socket /etc/systemd/system
-COPY .buildkite/config/tinysshd@.service /etc/systemd/system
-RUN systemctl enable tinysshd.socket
-COPY .buildkite/config/blockchain_limits.conf /etc/security/limits.d
+EXPOSE 14543
```

Comment:
That is this port?

Reply:
Thats the blockchain traffic port we talked about exposing